src/qt.ts

```diff
@@ -46,7 +46,11 @@ export async function findQtRootDirViaPathEnv(): Promise<string> {
         const paths = PATH.split(splitter);
         const exeExtension = tools.exeExtension();
         const mocFilenameOnly = `qmake${exeExtension}`;
-        const mocPath = await searchFileInDirectories(paths, [mocFilenameOnly]);
+        let filesnames = [mocFilenameOnly];
+        if(process.platform == "linux" && !await tools.fileExists("/etc/apt")) {
+            filesnames = ["qmake-qt5", "qmake-qt6"];
```

Comment:
* Any special reason why you also check `/etc/apt`?
* I would suggest to extend the `filesnames` variable, so that it contains `mocFilenameOnly` + the 2 new variants
  Just for the rare case that `qmake` will be just named `qmake`

Reply:
I want to deal with the CentOS/RedHat case separately, and the above code has vulnerabilities. I will modify it to use a `filenames`